lib/triggers/receive.js

```diff
@@ -23,7 +24,7 @@ exports.process = async function receive(msg) {
       msg.body._additionalUrlPath = msg.additionalUrlPath;
     }
 
-    this.logger.info('Updated body: \n', JSON.stringify(msg.body));
+    this.logger.debug('Updated body: \n', JSON.stringify(msg.body));
```

Comment:
IMO we must never print data, even in debug level.

Reply:
But what if customer wants to debug the incoming data?

Reply:
Next time some other developer will change the level to info for debugging purpose and will forget to change it back and so introduce the printing of data back to production

Reply:
> But what if customer wants to debug the incoming data?

The customer can't change the log level anyway. It can only be done by developer. If done, it is done for all customers using the component. That's why printing of data must be forbidden.

Reply:
would have been nice to switch on/off this per contract or workspace.

Reply:
Yes, but we don't have this possibility atm. We need to fill a product issue for that. Until it is there, we need to forbid logging data.

Reply:
> IMO we must never print data, even in debug level.
Strongly disagree.

Reply:
From perspective of a component developer I would also disagree. But a customer who cares about data protection does disagree with the developer. It's about our compliance.

Reply:
@jhorbulyk since we agreed with @juhrmacher on the policy for logging data in components may I ask you to adapt your PR by removing the logging of data completely?